Meta/check-style.py

```diff
@@ -54,6 +54,9 @@
     '.moc',
 ]
 
+# We check for and disallow any comments linking to the single-page HTML spec because it takes a long time to load.
+SINGLE_PAGE_HTML_SPEC_LINK = re.compile('//.*https://html\\.spec\\.whatwg\\.org/#')
```

Comment:
Using a regex seems like overkill here. Could we make this a string instead, then search for it using `if SINGLE_PAGE_HTML_SPEC_LINK in file_content`?

Reply:
No, because that would not catch comments like `// bla bla foo bar (see https://html\\.spec\\.whatwg\\.org/#foo-bar )`